SU-SWS/stanford_paragraph_types
# Earth 16 fw variants

# READY FOR REVIEW

# Summary
- New Paragraph types:
  - Highlight Card
- Refactored Full width banner to use paragraphs instead of field groups.
- No upgrade path has been created. Not sure if its needed this early.

# Needed By (Date)
- EOS (End of sprint)

# Urgency
- Kinda high due to such a big refactor.

# Steps to Test

1. Checkout branch & checkout EARTH_16-fw-variants branch of [stanford_components](https://github.com/SU-SWS/stanford_components/pull/45)
2. Enable/feature import stanford_paragraph_fw_banner & stanford_paragraph_highlight_card
3. create complex page with full width banner
4. populate all fields and save
5. output should be pretty similar to the patterns/section_highlight_banner page (needs some field templates as in https://github.com/SU-SWS/stanford_components/pull/48)

# See Also
- [stanford_components theme suggestions](https://github.com/SU-SWS/stanford_components/pull/48)
- [stanford_components patterns templates](https://github.com/SU-SWS/stanford_components/pull/45)
